builders/interoperability/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -541,7 +540,7 @@ const response = await from(polkadot).get(
 |       `asset`        |                   被转移的[资产](#assets)                    |
 | `existentialDeposit` | [当前存在的存款](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-#:~:text=On%20the%20Polkadot%20network%2C%20an,the%20Existential%20Deposit%20(ED).){target=_blank}，或是一个地址需要持有 以被定义为存在的最小数量，否则将返回`0n` |
 |        `min`         |                        最小可转移数量                        |
-|    `moonChainFee`    | 支付Moonbeam的XCM费用所需的[资产](#assets)和金额。如果与要转移的`asset`不同，则费用将在要转移的`asset`之外发送到该资产中 |
+|    `moonChainFee`    | 支付Moonbeam的XCM费用所需的[资产](#assets)和金额。如果与要转移的`asset`不同，则费用将在要转移的`asset`之外发送到该资产中(as of [v0.4.0](https://github.com/PureStake/xcm-sdk/releases/tag/v0.4.0){target=_blank}) |
```

Comment:
@wuxinxin2000 can you please check this over and provide a translation for "as of"?

English version:

```
the [asset](#assets) and amount that is needed to pay for Moonbeam's XCM fees. <br> If different from the `asset` to be transferred, the fee will be sent in this <br> asset in addition to the `asset` to be transferred (as of [v0.4.0](https://github.com/PureStake/xcm-sdk/releases/tag/v0.4.0){target=_blank})
```